content/docs/command-reference/add.md

```diff
@@ -151,6 +151,9 @@ not.
 - `--desc <text>` - user description of the data (optional). This doesn't affect
   any DVC operations.
 
+- `-o`, `--output` - adds the `<target>` file from outside the repo to the repo
+  at the specified `<path>`.
```

Comment:
Quick copy edit:
```suggestion
- `-o <path>`, `--output <path>` - adds a single external target (file or
  directory) to the specified `path`.
```

Reply:
But TBH I don't quite understand what this option does from this text.

Reply:
In any case (as mentioned in https://github.com/iterative/dvc/pull/5284#pullrequestreview-571029531) the same(?) option is being introduced separately, explained differently [in #2091](https://github.com/iterative/dvc.org/pull/2091/files#diff-e0a7914717a515b54fca157272d09319399f56081e915137ea2e5175d4417319R177-R178):

> - `-o <path>`, `--out <path>` - destination path for the transferred data (can
  only be used with `--to-remote`).

Is it the same concept?